doc/api/pvalue.rst

```diff
@@ -3,11 +3,11 @@
 p-value
 ==============================================
 
-.. function:: pvalue(test::HypothesisTest; tail=:both)
+.. function:: pvalue(test::HypothesisTest; tail=tail(test))
 
     Compute the p-value for a given significance test.
 
-    If ``tail`` is ``:both`` (default), then the p-value for the
```

Comment:
Would be nice to mention that the default tail depends on the specific test, and give a list of tests with their default.